run/navigation/demo_log_fake_navigation.cpp

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
This file is almost the same as `demo_fake_navigation.cpp`, right? Do we really need both?

Reply:
Yeah they're very similar, but one utilises the imu logger. I suppose they could both go in the same file, and then based on a flag one or the other would be called. They don't really clutter anything up atm because there is just a directory in which both of them are in, so I don't see it as anything urgent to change.